envoy-control-core/src/test/kotlin/pl/allegro/tech/servicemesh/envoycontrol/snapshot/resource/listeners/filters/rbac/RBACFilterFactoryTest.kt

```diff
@@ -86,6 +87,20 @@ internal class RBACFilterFactoryTest : RBACFilterFactoryTestUtils {
         StatusRouteProperties()
     )
 
+    private val rbacFilterFactoryWithDefaultAndCustomClientsLists = RBACFilterFactory(
```

Comment:
Could you put two elements to every list, to make sure that all elements are added?

Reply:
added